build.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
Could you check using [shellcheck](https://www.shellcheck.net/)?

Reply:
Thanks! Just updated.